network/wsNetwork_test.go

```diff
@@ -4136,82 +4136,95 @@ func TestRefreshRelayArchivePhonebookAddresses(t *testing.T) {
+				"r2archive.algorand-<network>.network", "r3archive.algorand-<network>.network"} {
+				var recordSub = strings.Replace(record, "<network>", string(netA.NetworkID), -1)
+				primaryArchiveResolvedRecords = append(primaryArchiveResolvedRecords, recordSub)
+				secondaryArchiveResolvedRecords = append(secondaryArchiveResolvedRecords, strings.Replace(recordSub, "network", "net", -1))
 			}
 
-			if service == "archive" && protocol == "tcp" && name == primarySRVBootstrap {
-				return primaryArchiveResolvedRecords, nil
-			} else if service == "archive" && protocol == "tcp" && name == backupSRVBootstrap {
-				return secondaryArchiveResolvedRecords, nil
+			// Mock the SRV record lookup
+			netA.resolveSRVRecords = func(service string, protocol string, name string, fallbackDNSResolverAddress string,
+				secure bool) (addrs []string, err error) {
+				if service == "algobootstrap" && protocol == "tcp" && name == primarySRVBootstrap {
+					return primaryRelayResolvedRecords, nil
+				} else if service == "algobootstrap" && protocol == "tcp" && name == backupSRVBootstrap {
+					return secondaryRelayResolvedRecords, nil
+				}
+
+				if service == "archive" && protocol == "tcp" && name == primarySRVBootstrap {
+					return primaryArchiveResolvedRecords, nil
+				} else if service == "archive" && protocol == "tcp" && name == backupSRVBootstrap {
+					return secondaryArchiveResolvedRecords, nil
+				}
+
+				return
 			}
 
-			return
-		}
+			relayPeers := netA.GetPeers(PeersPhonebookRelays)
+			assert.Equal(t, 0, len(relayPeers))
 
-		relayPeers := netA.GetPeers(PeersPhonebookRelays)
-		assert.Equal(t, 0, len(relayPeers))
+			archivePeers := netA.GetPeers(PeersPhonebookArchivers)
+			assert.Equal(t, 0, len(archivePeers))
 
-		archivePeers := netA.GetPeers(PeersPhonebookArchivers)
-		assert.Equal(t, 0, len(archivePeers))
+			netA.refreshRelayArchivePhonebookAddresses()
 
-		netA.refreshRelayArchivePhonebookAddresses()
+			relayPeers = netA.GetPeers(PeersPhonebookRelays)
 
-		relayPeers = netA.GetPeers(PeersPhonebookRelays)
+			assert.Equal(t, 3, len(relayPeers))
+			relayAddrs := make([]string, 0, len(relayPeers))
+			for _, peer := range relayPeers {
+				relayAddrs = append(relayAddrs, peer.(HTTPPeer).GetAddress())
+			}
 
-		assert.Equal(t, 3, len(relayPeers))
-		relayAddrs := make([]string, 0, len(relayPeers))
-		for _, peer := range relayPeers {
-			relayAddrs = append(relayAddrs, peer.(HTTPPeer).GetAddress())
-		}
+			assert.ElementsMatch(t, primaryRelayResolvedRecords, relayAddrs)
 
-		assert.ElementsMatch(t, primaryRelayResolvedRecords, relayAddrs)
+			archivePeers = netA.GetPeers(PeersPhonebookArchivers)
 
-		archivePeers = netA.GetPeers(PeersPhonebookArchivers)
+			if refreshTestConf.EnableBlockServiceFallbackToArchiver {
+				// For the time being, we do not dedup resolved archive nodes
+				assert.Equal(t, len(primaryArchiveResolvedRecords)+len(secondaryArchiveResolvedRecords), len(archivePeers))
```

Comment:
This isn't a change is it, just removing the magic number?

Reply:
I saw an opportunity to fix a magic number that probably should not have been there.